archunit/src/test/java/com/tngtech/archunit/core/domain/JavaAccessTest.java

```diff
@@ -23,6 +26,9 @@ public void when_the_origin_is_an_inner_class_the_toplevel_class_is_displayed_as
                 .inLineNumber(7);
 
         assertThat(access.getDescription()).contains("(SomeClass.java:7)");
+        Dependency dependency = getOnlyElement(access.toClassDependencies());
+        assertThatType(dependency.getTargetClass()).as("target class").isEqualTo(access.getTargetOwner());
+        Assertions.assertThat(dependency.getDescription()).as("description").isEqualTo(access.getDescription());
```

Comment:
I'd move the test for `access.toClassDependencies()` to a separate test, as it's not really related to `when_the_origin_is_an_inner_class_the_toplevel_class_is_displayed_as_location`.

Reply:
Done